pages/1.13/gui/secrets/index.md

```diff
@@ -10,6 +10,67 @@ model: /1.13/data.yml
 ---
 
 You can manage secrets and certificates from the Secrets page.
-![Secrets](/1.13/img/GUI-Secrets-Secrets_View_With_Secrets-1_12.png)
 
+![Secrets](/1.13/img/GUI-Secrets-Secrets_View_With_Secrets-1_12.png)
 Figure 1 - Secrets page
+
+
+For complete details on creating and managing Secrets, see the [Secrets](/1.13/security/ent/secrets) documentation.
+
+<p class="message--important"><strong>IMPORTANT: </strong>The maximum file size for a secret is approximately 1 MB, subtracting approximately 1 KB for the secret store metadata.</p>
+
+# Creating key-value pair secrets 
+
+1. Log in to the DC/OS UI as a user with the `dcos:superuser` permission.
+
+1. Open the **Secrets** tab.
+
+1. Click the **+** icon in the top right.
+
+    ![New Secret](/1.13/img/new-secret.png)
```

Comment:
Would you mind updating the screenshot please?

![Screenshot 2019-08-06 at 13 00 58](https://user-images.githubusercontent.com/186223/62534837-41a60200-b84a-11e9-874c-72c037a59658.png)